### DISCUSSION TITLE
Привет Россия

### DISCUSSION THREAD

**Discussion by Benman2785** [JohnImril/diablo_web#60] [General]
i always wanted to visit Russia, yet the nearest i got was Tallinn.
thank you for your good work.

i hope i dont get on your nerves with my many messages :)
only friday and then we have weekend - enjoy

**Comment by JohnImril** [comment]
Hi, I'm really glad someone is interested in the project. And I want to help the community as much as I can, there are a lot of interesting people and projects here. If you have questions about frontend development (and these are not necessarily just my projects), then feel free to ask them, and I will try to answer them. Maybe someday, someone I helped will be able to recommend me for work. And why did I choose Diablo? It was my first game in my life, and so I wanted to leave some mark on its support and development. It is important to me.

**Comment by JohnImril** [comment]
Your interest keeps me going.

**Comment by Benman2785** [comment]
Thank you for you nice words. I am also interested in your work as Diablo was the first game i played with my friends in Multiplayer - all those good memories :)

About work - if you are willing to move and have some certification you can try to look for jobs in Germany -> currently here are many opportunities.

Also a big thank you for creating your https://github.com/JohnImril/diablo_web_simple repo -> this helps me a lot

**Reply by JohnImril** [reply]
Hi, what certificates are valued? And how important do you think it is to have a LinkedIn profile or are all job postings mostly on more specialized sites?

**Reply by Benman2785** [reply]
Hey, sorry for the late answer:

80% of coding is Java, 15% is Rust, 5% is C/C++
if you are going the web-dev-route: js, php and css

LinkedIn is a nice to have -> most jobs are found here (use translator):
https://web.arbeitsagentur.de/berufenet/beruf/7856#arbeitsmarkt_arbeitsStellenmarkt_stellenboersen
(look in Stellen- und Bewerberbörsen)

most jobs require German C1 and english // international companies may only use english

**Reply by JohnImril** [reply]
Thanks a lot!!!

**Comment by Benman2785** [comment]
thank you :)
